release-guide.md

```diff
@@ -0,0 +1,49 @@
+# Release Guide
+
+## Requirements
+- JDK 8
+- Protobuf 2.5.0
+- GNU Sed 4.2.2 + (for generating new apache-casssandra package using `build-cassandra-bin.bash`)
+
+## Clone the repo
+```
+$ git clone git@github.com:mesosphere/dcos-cassandra-service.git
+```
+
+## Build instructions
+```
+$ ./gradlew clean build
+```
+
+## Updating apache-cassandra binary package
+
+We need to update the vanilla apache-cassandra binary package for following reasons:
+
+1. Make JMX_PORT configurable
+2. Copy reporter-config JARs into lib directory for metric reporting:
+
+To generate a new apache-cassandra binary package:
+
+1. Execute following
+
+  ```bash
+  $ ./build-cassandra-bin.bash
+  ```
+See [build-cassandra-bin.bash](https://github.com/mesosphere/dcos-cassandra-service/blob/master/build-cassandra-bin.bash) for more details on usage.
+2. Upload the `/path/to/dcos-cassandra-service/cassandra-bin-tmp/apache-cassandra-A.B.C-bin-dcos.tar.gz` to the designated release location.
+3. Update the universe's [resource.json](https://github.com/mesosphere/dcos-cassandra-service/blob/master/universe/resource.json) with new URL if necessary.
+
+## Releasing DC/OS Cassandra service
+
+1. Make necessary chanages to [dcos-cassandra-service/universe](https://github.com/mesosphere/dcos-cassandra-service/tree/master/universe)
+2. Prepare RELEASE NOTES
+3. Cut a tag with following format: `X.Y.Z-A.B.C`, where `X.Y.Z` is the version of service and `A.B.C` is the version of underlying Cassandra
+
+  ```
+  git tag X.Y.Z-A.B.C
+  git push origin X.Y.Z-A.B.C
+  ```
+
+4. Trigger the [0-trigger-tag](https://jenkins.mesosphere.com/service/jenkins/view/Infinity/job/cassandra/job/0-trigger-tag/) job with the tag version created in step 3.
```

Comment:
The plumbing between `0-trigger-tag` and `release-to-universe` is missing right now. We can certainly improve it.